src/rpdk/core/contract/resource_client.py

```diff
@@ -161,13 +160,6 @@ def __init__(
         self._inputs = inputs
         self._timeout_in_seconds = int(timeout_in_seconds)
 
-    def _get_partition(self):
```

Comment:
for handlers which use partition. How will that be handled?

Reply:
we infer on the handler side: https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/blob/master/src/cloudformation_cli_python_lib/utils.py#L135

https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/a4bf268462367ea43c6a3f5269a9ede42d1b2a89/python/rpdk/java/templates/generate/HandlerWrapper.java#L127-L142

Reply:
thanks for the info. It seems that the prod entry should pick up the partition information